test/public/overview.test.js

```diff
@@ -53,19 +52,21 @@ module.exports = function () {
         });
 
         pti.write([...jsCoverage, ...cssCoverage]);
+        await server.close();
         await browser.close();
     });
 
     it('loads the page successfully', async () => {
-        const response = await page.goto(`http://localhost:${PORT}`);
+        const response = await page.goto(URL);
         // We expect the page to return the correct status code, making sure the server is running properly
         assert.equal(response.status(), 200);
         const title = await page.title();
         // We expect the page to return the correct title, making sure there isn't another server running on this port
         assert.equal(title, 'AliceO2 Logbook 2020');
-    });
+    }, 2000);
 
     it('can filter logs dynamically', async () => {
+        await page.waitFor(1000);
```

Comment:
Why wait at the start of the method?

Reply:
Changed the waiting positions, this one I had put in place to make sure the page was loaded before grabbing any elements.